ProjectLighthouse.Servers.GameServer/Extensions/ControllerExtensions.cs

```diff
@@ -175,6 +175,9 @@ void ParseLbp3Query(string key, Action allMust, Action noneCan, Action dontCare)
 
         if (!queryBuilder.GetFilters(typeof(CrossControlFilter)).Any())
             queryBuilder.AddFilter(new ExcludeCrossControlFilter());
+        
+        if (controller.Request.Path.Value != null && controller.Request.Path.Value.EndsWith("lbp2luckydip"))
+            queryBuilder.AddFilter(new LockedSlotFilter());
```

Comment:
```suggestion
```
I think since this method is mostly for filtering based on the query params of the request, this would be better suited in the endpoint controllers for lucky dip. Specifically, in [LuckyDipSlots()](https://github.com/LBPUnion/ProjectLighthouse/blob/9ac74312ce44f2bc47eb447d81c305fc83d17e94/ProjectLighthouse.Servers.GameServer/Controllers/Slots/SlotsController.cs#L265) for LBP2 and in [LuckyDipCategory.cs](https://github.com/LBPUnion/ProjectLighthouse/blob/9ac74312ce44f2bc47eb447d81c305fc83d17e94/ProjectLighthouse.Servers.GameServer/Types/Categories/LuckyDipCategory.cs#L20) for LBP3. You should be able to do `queryBuilder.AddFilter(new LockedSlotFilter())` for both of them.

Reply:
for sure, this is the less jank implementation i was looking for ! for some reason it didnt occur to me that i could access the querybuilder outside of filterfromrequest